tests/cuda/test_basic.cu

```diff
@@ -0,0 +1,50 @@
+/**
+ * TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#include <gtest/gtest.h>
+
+TEST(CUDABasic, DeviceCount) {
+    int nDev;
```

Comment:
Please set default values to every variable. Having just been through 2 weeks of programming tutorials, we should not ignore one of the rules that we teach to all newcomers...